contracts/remappings.txt

```diff
@@ -6,3 +6,9 @@ hardhat/=node_modules/hardhat/
 @eth-optimism/=node_modules/@eth-optimism/
 @scroll-tech/=node_modules/@scroll-tech/
 @zksync/=node_modules/@zksync/
+
+@aave/=foundry-lib/gho-core/lib/aave-token/
+@aave-gho-core/=foundry-lib/gho-core/src/contracts/
+@aave/core-v3/=foundry-lib/gho-core/lib/aave-v3-core/
+@aave/periphery-v3/=foundry-lib/gho-core/lib/aave-v3-periphery/
```

Comment:
these two are needed here because they are dependencies of `@aave-gho-core`?

Reply:
yes